packages/TorneloScoresheet/src/components/ChessBoard/style.ts

```diff
@@ -0,0 +1,18 @@
+import { StyleSheet } from 'react-native';
+import { colours } from '../../style/colour';
+
+export const styles = StyleSheet.create({
+  board: {
+    display: 'flex',
+    flexDirection: 'column',
+    height: 80 * 8,
+    width: 80 * 8,
```

Comment:
Let's make this a constant that's exported from `ChessSquare.tsx` and imported here. That way, if we want to scale this, we can change it in one file.

`ChessSquare/style.tsx`:
```ts
export const CHESS_SQUARE_SIZE = 80;
```